point/nexstar.py

```diff
@@ -377,9 +377,10 @@ def set_time(self, timestamp=None):
         disabled. This effectively means that the hand controller clock will be set to UTC.
 
         Args:
-            timesamp (int): A Unix timestamp (seconds since 1 Jan 1970 in UTC minus leap seconds).
+            timestamp (int): A Unix timestamp (seconds since 1 Jan 1970 in UTC minus leap seconds).
                 If omitted, the time will be obtained from the clock of the machine running this
                 Python program.
+                :param timestamp:
```

Comment:
Why this addition?

Reply:
Not sure how that got there, will remove.